students/arun_nalla/session05/Lesson5_Mailroom_Part3.py

```diff
@@ -0,0 +1,99 @@
+#!usr/bin/env python
+
+donors_list = [('Aron King', [1, 2]), ('Becky Patty', [3000, 1000]),
+               ('Charlie Lee', [666, 222]), ('Thomas Dave', [202, 90, 450]), ('Nancy Crow', [100, 200, 300])]
+# input from Mairoom_Part1. Convert into a dict
+
+donors_dict = {x: y for x, y in donors_list}
```

Comment:
I must have overlooked this initialization code. No need to create a separate list just for dict initialization. Can do the initialization directly:
```
donors_dict = { 'Aron King': [1, 2], 'Becky Patty': [3000, 1000], ... }
```
I might have already mentioned this... I'll remember that and stop making any more repeated comments... :)